tests/filecheck/mlir-conversion/with-mlir/dialects/omp/ops.mlir

```diff
@@ -130,136 +130,150 @@ builtin.module {
     }) : () -> ()
     func.return
   }
+  func.func @omp_target(%dep : memref<6xf32>, %dev : i64, %host : i32, %if : i1, %p1 : memref<10xi8>, %p2 : f64, %tlimit : i32) {
+    "omp.target"(%dep, %dev, %host, %if, %p1, %p2, %tlimit) <{operandSegmentSizes = array<i32: 0, 0, 1, 1, 0, 1, 1, 0, 0, 0, 2, 1>, in_reduction_syms = [@rsym1, @rsym2], private_syms = [@psym1, @psym2], nowait, bare, depend_kinds = [#omp<clause_task_depend(taskdependinout)>]}> ({
+    ^0(%b_host : i32, %b_p1 : memref<10xi8>, %b_p2 : f64):
+      "omp.terminator"() : () -> ()
+    }) : (memref<6xf32>, i64, i32, i1, memref<10xi8>, f64, i32) -> ()
+    func.return
+  }
 }
 
 // CHECK:       builtin.module {
-// CHECK-NEXT:    func.func @omp_parallel(%arg37 : memref<1xi32>, %arg38 : i1, %arg39 : i32) {
-// CHECK-NEXT:      "omp.parallel"(%arg37, %arg37, %arg38, %arg39) <{operandSegmentSizes = array<i32: 1, 1, 1, 1, 0, 0>, proc_bind_kind = #omp<procbindkind spread>}> ({
-// CHECK-NEXT:        "omp.parallel"(%arg37, %arg37, %arg39) <{operandSegmentSizes = array<i32: 1, 1, 0, 1, 0, 0>}> ({
+// CHECK-NEXT:    func.func @omp_parallel(%arg47 : memref<1xi32>, %arg48 : i1, %arg49 : i32) {
```

Comment:
do you know why there are all these changes? Seems weird to me, I would expect all of these to be isolated from above